JuliaStream.jl/src/CUDAStream.jl

```diff
@@ -0,0 +1,152 @@
+include("Stream.jl")
+using CUDA
+
+const CuData = StreamData{T,CuArray{T}} where {T}
+const TBSize = 1024::Int
+const DotBlocks = 256::Int
+
+function devices()
+  return !CUDA.functional(false) ? [] :
```

Comment:
Minor nitpick: make the function type-stable
```suggestion
  return !CUDA.functional(false) ? String[] :
```